pkg/fission-cli/spec.go

```diff
@@ -343,6 +343,27 @@ func (fr *FissionResources) validate() error {
 			packages[mapKey(pkgMeta)] = true
 		}
 
+		client := util.GetApiClient(c.GlobalString("server"))
+		for _, cm := range f.Spec.ConfigMaps {
+			c, err := client.ConfigMapGet(&metav1.ObjectMeta{
```

Comment:
Since the parameter pass-in is already named `c`, how about changing the variable name here to prevent confusion?